content/en/network_monitoring/dns/_index.md

```diff
@@ -42,29 +42,37 @@ Are you looking for Network Device Monitoring instead? See the [NDM setup instru
 
 ## Queries
 
-Use the source and destination search bars at the top of the page to query for dependencies between a client (_source_), which makes the DNS request, and a DNS server (_destination_), which responds to the DNS request. The destination port is automatically scoped to DNS port 53 so that all resulting dependencies match this (client → DNS server) format.
+Use the search bar at the top of the page to query for dependencies between a client, which makes the DNS request, and a DNS server, which responds to the DNS request. The destination port is automatically scoped to DNS port 53 so that all resulting dependencies match this (client → DNS server) format.
```

Comment:
```suggestion
Use the search bar at the top of the page to query for dependencies between a client (which makes the DNS request) and a DNS server (which responds to the DNS request). The destination port is automatically scoped to DNS port 53 so that all resulting dependencies match this (client → DNS server) format.
```